src/pages/input-range.ts

```diff
@@ -123,10 +123,39 @@ function setPreview(values: Values): void {
   );
 }
 
+function areInputsValid() {
+  const getTrackColor = getColorInput1(attribute);
+  const getThumbColor = getColorInput2(attribute);
+
+  const areColorInputsValid = getTrackColor.value && getThumbColor.value;
+
+  const trackCheckBox = getCheckbox(`${attribute}-track`);
+  const getTrackRadius = getRadiusInput(`${attribute}-track`);
+
+  const isTrackRadiusInputValid = trackCheckBox.checked
+    ? getTrackRadius.value !== ''
+    : true;
+
+  const thumbCheckBox = getCheckbox(`${attribute}-thumb`);
+  const getThumbRadius = getRadiusInput(`${attribute}-thumb`);
+
+  const isThumbRadiusInputValid = thumbCheckBox.checked
+    ? getThumbRadius.value !== ''
+    : true;
+
```

Comment:
your issue isn't to check if they're valid or not, it's to check if they exist, not part of your issue, remove it

Reply:
We can resolve this conversation to avoid duplicacy.
Ref : https://github.com/Dun-sin/Code-Magic/pull/497#discussion_r1360951993